back/src/forms/compat.ts

```diff
@@ -11,9 +11,9 @@ import { Bsdd } from "./types";
  * @returns
  */
 export function simpleFormToBsdd(
-  form: Form & { transportSegments?: TransportSegment[] }
+  form: Form & { transporters?: BsddTransporter[] }
```

Comment:
```suggestion
  form: Form & { transporters: BsddTransporter[] }
```

Il faudrait que `transporters` ne puisse pas être undefined ici non ? Sinon les calls genre `transporter.transporterCompanyName` en dessous peuvent fail. Je suis étonné que Typescript ne râle pas 🤔